qpc_tools/messages.py

```diff
@@ -13,15 +13,29 @@
 """QPC Tools messages for translation."""
 VERBOSITY_HELP = 'Verbose mode. Use up to -vvvv for more verbosity.'
 
-INSTALL_HOME_DIR_HELP = 'The home directory for the Quipucords applications '\
-    'to store data (defaults to ~/quipucords)'
 
 ALL_INSTALLATION_SUCCESSFUL = 'Installation of both server and CLI was successful'
+ALL_INSTALL_OFFLINE_HELP = 'Perform an offline CLI and server installation (defaults to false)'
+ALL_INSTALL_OFFLINE_FILES_HELP = 'Specify the path to the CLI and server offline files'
+ALL_INSTALL_HOME_DIR_HELP = 'The home directory for the Quipucords application data '\
+    '(defaults to ~/quipucords)'
 
 SERVER_INSTALLATION_SUCCESSFUL = 'Installation of server was successful'
+SERVER_INSTALL_OFFLINE_HELP = 'Perform an offline Server installation (defaults to false)'
+SERVER_INSTALL_OFFLINE_FILES_HELP = 'Specify the path to the server offline files'
+SERVER_INSTALL_VERSION_HELP = 'Specify the QPC Server version to install '\
+    '(defaults to latest release)'
+SERVER_INSTALL_PORT_HELP = 'Port number of the server (defaults to 9443)'
```

Comment:
Is it necessary to duplicate this message since we also define it for the cli? Only the variable name is changed.

Reply:
Fixed